backend/models/issues.py

```diff
@@ -53,8 +53,9 @@ class Issue(db.Model):
     user_id = db.Column(db.ForeignKey(u'users.id'), index=True)
     category_id = db.Column(db.ForeignKey(
         u'category.id'), nullable=False, index=True)
+    locationX = db.Column(db.Float)
+    locationY = db.Column(db.Float)
```

Comment:
I'm not sure it's a good names for these fields. First at all the are not follow naming style, secondly names are not reflect what they are store. I'd suggest to use something like `loacation_lon` and `location_lat`.